.circleci/config.yml

```diff
@@ -94,20 +103,48 @@ jobs:
           paths:
             - yarn
   test-node8:
```

Comment:
Can we name them `test-linux-node8` etc? They will mirror nicely `test-osx-node8`.

Reply:
Great idea! Done.